crates/rspack_plugin_devtool/src/lib.rs

```diff
@@ -274,6 +756,24 @@ impl Plugin for EvalSourceMapDevToolPlugin {
     "rspack.EvalSourceMapDevToolPlugin"
   }
 
+  async fn compilation(
+    &self,
+    _args: CompilationArgs<'_>,
+    _params: &CompilationParams,
+  ) -> PluginCompilationHookOutput {
+    // TODO: Temporarily use `devtool` to pass source map configuration information
+    let mut devtool = _args.compilation.options.devtool.lock().await;
+    devtool.add_source_map();
+    devtool.add_eval();
+    if !self.columns {
+      devtool.add_cheap();
+    }
+    if self.no_sources {
+      devtool.add_no_sources();
+    }
```

Comment:
also need to handle "module" (eval-cheap-module-source-map)

Reply:
The decision might be made in `packages/rspack/src/rspackOptionsApply.ts`.